src/pages/Checkout/CheckoutPage.vue

```diff
@@ -493,6 +498,21 @@ export default {
 		// TODO: Implement check against contentful setting
 		// to signify if holiday mode is enabled
 
+		// VUE-1725 Async Checkout Mutation Rollout
+		const asyncCheckoutExp = this.apollo.readFragment({
+			id: `Experiment:${ASYNC_CHECKOUT_EXP}`,
+			fragment: experimentVersionFragment,
+		}) || {};
+
+		this.asyncCheckoutActive = asyncCheckoutExp?.version === 'b';
+		if (asyncCheckoutExp?.version !== null) {
+			this.$kvTrackEvent(
+				'Basket',
+				'EXP-VUE-1725-Aug2023',
+				this.asyncCheckoutExpVersion,
+			);
+		}
```

Comment:
I've removed the gates here for initialization to ensure this takes affect from an empty basket state. The tracking is fine to fire at all times.